mongo/doc.go

```diff
@@ -113,7 +114,8 @@
 //    }
 //    aeo.SetExtraOptions(mongocryptdOpts)
 // To specify a process URI for mongocryptd, the "mongocryptdURI" option can be passed in the ExtraOptions map as well.
-// More information about mongocryptd will soon be available from the official documentation.
+// More information about mongocryptd will soon be available from the official documentation. See the
```

Comment:
Looks like it's available now: https://docs.mongodb.com/manual/core/security-client-side-encryption/index.html#field-level-encryption-mongocryptd

Reply:
Updated the docs in the new commit.